### ISSUE TITLE
Suggest subjects and tags where required

### ISSUE THREAD

**Issue by nemobis** [dissemin/dissemin#540]
To speed up and ease the deposit into HAL and OSF, it would be good to prefill those fields with some reasonable suggestions and let authors decide what to keep.

http://annif.org/ by the National Library of Finland can be used for this purpose and was in fact already experimentally used by [Ari](www.opendimension.org/) in [JYU](https://osc.jyu.fi/fi/blogi/julkaisuarkisto-jyx-uudistuu) for the students' thesis deposit.

A simple approach is to just rely on the data and model they already have and map the [YSO top subjets](http://finto.fi/yso/en/groups) to ours (see [spreadsheet with an initial map](https://wikimediaitalia.nws.netways.de/index.php/s/3Jrb85CHFrDM2of)).

A more robust solution would be to actually download works from OSF and HAL and train Annif against the content of each subject, to create a custom "project".

**Comment by wetneb** [comment]
Great! Note that we already have http://haltopics.dissem.in:6377/ for HAL subjects.

**Comment by nemobis** [comment]
Indeed, I've just noticed it in `deposit/hal/protocol.py`. :) I'll try with OSF.

**Comment by nemobis** [comment]
This is a subtask (or duplicate) of https://github.com/dissemin/dissemin/issues/465
